File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/commands/SeatAssignments/CreateSeatAssignments.cs
using System;
using ENode.Commanding;

namespace Registration.Commands.SeatAssignments
{
    public class CreateSeatAssignments : Command<Guid>
    {
        public CreateSeatAssignments() { }
        public CreateSeatAssignments(Guid orderId) : base(orderId) { }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/registration/commands/SeatAssignments/CreateSeatAssignments.java
package com.microsoft.conference.common.registration.commands.SeatAssignments;

import com.enodeframework.commanding.Command;

public class CreateSeatAssignments extends Command<String> {
    public CreateSeatAssignments() {
    }

    public CreateSeatAssignments(String orderId) {
        super(orderId);
    }

}
